prime-dotnet-webapi/Models/Statuses/SiteStatus.cs

```diff
@@ -0,0 +1,35 @@
+using System;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Text.Json.Serialization;
+
+namespace Prime.Models
+{
+    [Table("SiteStatus")]
+    public class SiteStatus : BaseAuditable
+    {
+        [Key]
+        public int Id { get; set; }
+
+        public int SiteId { get; set; }
+
+        // [JsonIgnore]
+        // public Site Site { get; set; }
```

Comment:
Is this commented code intentional?

Reply:
With the code enabled, I get this, despite the `[JsonIgnore]`?
![image](https://user-images.githubusercontent.com/7586665/123289868-0d836b00-d4ce-11eb-98f3-352092eff492.png)

My use of the class doesn't seem impeded without the property?

Reply:
wrong Json import.
using Newtonsoft.Json;

Reply:
The greatest accidental haiku of all time